packages/ui-utils/src/ms.ts

```diff
@@ -36,21 +36,20 @@ import { parseUnit } from './parseUnit'
  *  - '100s'
  *  - '20ms'
  *
- * @param {String} val
+ * @param {String|Number} val
  * @returns {Number} Returns numerical representation of milliseconds
  */
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'val' implicitly has an 'any' type.
-function ms(val) {
+function ms(val: number | string): number {
+  // TODO: throw an error if val is not number or string
   if (!val || typeof val === 'number') {
-    return val
+    return val as number
   }
```

Comment:
again some weird code that allows to return edge cases like null or undefined. I've left it this way to not change functionality